src/widgets/latest-transactions/LatestTransactions.vue

```diff
@@ -22,7 +22,7 @@
         class="latest-transactions__row"
       >
         <TransactionStatus
-          :committed="transaction.committed"
+          :committed="!!transaction.rejection_reason"
```

Comment:
You see, in the previous form there was a ready-to-use `committed` flag, but now it's the same `!transaction.rejection_reason` in a lot of places. And it's so much easier to make a typo
```suggestion
          :committed="!transaction.rejection_reason"
```

Reply:
For now we are using TransactionDto type. Later when mapping transaction from dto will be applied we will rely on committed field. Anyway I will correct such expressions.